unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/VoiceChat/DCLVoiceChatController.cs

```diff
@@ -4,16 +4,15 @@
 using System;
 using System.Collections.Generic;
 using UnityEngine;
+using UnityEngine.Serialization;
 
 namespace DCL
 {
     public class DCLVoiceChatController : MonoBehaviour
     {
         [Header("InputActions")]
-        public InputAction_Hold voiceChatAction;
-
-        private InputAction_Hold.Started voiceChatStartedDelegate;
-        private InputAction_Hold.Finished voiceChatFinishedDelegate;
+        public InputAction_Hold voiceChatHoldAction;
+        public InputAction_Trigger voiceChatLeaveOnShortcutTrigger;
```

Comment:
```suggestion
        public InputAction_Trigger voiceChatToggleAction;
```
Would toggle be more suitable name for that?

Reply:
Sure, looks clearer. Changing!